RELEASING/README.md

```diff
@@ -79,27 +87,28 @@ often stretching over several weeks calendar time if votes don't pass, chances a
 the same terminal session won't be used for crafting the release candidate and the
 final release. Therefore, it's a good idea to do the following every time you
 work on a new phase of the release process to make sure you aren't releasing
-the wrong files/using wrong names:
+the wrong files/using wrong names. There's a script to help you set correctly all the
+necessary environment variables:
 
-```bash
-    # Set SUPERSET_VERSION to the release being prepared, e.g. 0.34.1.
-    export SUPERSET_VERSION=XX.YY.ZZ
-    # Set RC to the release candindate number. Replacing QQ below with 1
-    # indicates rc1 i.e. first vote on version above (0.34.1rc1)
-    export SUPERSET_RC=QQ
+```$bash
+# usage: set_release_env.sh <SUPERSET_VERSION> <SUPERSET_VERSION_RC> "<PGP_KEY_FULLNAME>"
+. ./set_release_env.sh 0.34.1 1 "YOUR FULL NAME HERE"
```

Comment:
Perhaps we should replace the hard-coded version params with bogus params (e.g. the XX.YY.ZZ QQ), potentially with real values in an example in a comment above, just to ensure someone doesn't accidentally use the wrong ones?